pattern-2/bosh-release/jobs/gateway/templates/ctl.erb

```diff
@@ -67,7 +67,7 @@ dpkg -s nfs-common >/dev/null 2>&1
 if [ $? -ne 0 ] ; then
   log_debug "Installing NFS Client"
   sudo apt-get update
-  sudo apt-get install nfs-common
+  sudo apt-get install nfs-common -y
```

Comment:
We need this option only if we change the stemcell. Shall we update the stemcell as well?